### ISSUE TITLE
Message: invalid element state: Element is not currently interactable and may not be manipulated

### ISSUE THREAD

**Issue by Airwave1981** [Second-Hand-Friends/kleinanzeigen-bot#74]
Hi all,

while creating an ad short before or when adding pictures I get following error:

```
←[1;30m[ERROR]←[0m ←[31mInvalidElementStateException: Message: invalid element state: Element is not currently interactable and may not be manipulated
  (Session info: MicrosoftEdge=101.0.1210.47)
Stacktrace:
Backtrace:
        Microsoft::Applications::Events::EventProperties::unpack [0x00007FF6A342CC52+24658]
        Microsoft::Applications::Events::time_ticks_t::time_ticks_t [0x00007FF6A336D342+539922]
        Ordinal0 [0x00007FF6A2E70F05+659205]
        Ordinal0 [0x00007FF6A2E744FA+673018]
        Ordinal0 [0x00007FF6A2E7427B+672379]
        Ordinal0 [0x00007FF6A2E74630+673328]
        Ordinal0 [0x00007FF6A2EA2F6E+864110]
        Ordinal0 [0x00007FF6A2EC769A+1013402]
        Ordinal0 [0x00007FF6A2E9FF34+851764]
        Ordinal0 [0x00007FF6A2EC79B0+1014192]
        Ordinal0 [0x00007FF6A2EDB9E7+1096167]
        Ordinal0 [0x00007FF6A2EC74B3+1012915]
        Ordinal0 [0x00007FF6A2E9E440+844864]
        Ordinal0 [0x00007FF6A2E9F848+849992]
        Microsoft::Applications::Events::EventProperty::EventProperty [0x00007FF6A31317D8+120584]
        Microsoft::Applications::Events::EventProperty::EventProperty [0x00007FF6A311CE1E+36174]
        Microsoft::Applications::Events::EventProperty::EventProperty [0x00007FF6A312001C+48972]
        Microsoft::Applications::Events::EventProperty::to_string [0x00007FF6A2F70716+30806]
        Microsoft::Applications::Events::time_ticks_t::time_ticks_t [0x00007FF6A3374611+569313]
        Microsoft::Applications::Events::time_ticks_t::time_ticks_t [0x00007FF6A3379044+588308]
        Microsoft::Applications::Events::time_ticks_t::time_ticks_t [0x00007FF6A337919D+588653]
        Microsoft::Applications::Events::time_ticks_t::time_ticks_t [0x00007FF6A338292E+627454]
        BaseThreadInitThunk [0x00007FF92D6D7034+20]
        RtlUserThreadStart [0x00007FF92E0E2651+33]
←[0m
```

I wanted to debug, but haven't debugged pdm projects yet.
Opened __init__.py in vscode, started debug but then get error No module named 'overrides' when trying to start debugging. (Wondering if should install pip overrides).
Maybe someone can point me to how I can debug this project suitable.

Thanks

**Comment by sebthom** [comment]
Maybe you find some useful instructions regarding vscode setup here https://pdm.fming.dev/#use-with-ide

**Comment by sebthom** [comment]
I have not encountered this myself. Maybe it is an issue with specific versions of the chrome driver. 

Also I found this https://github.com/seleniumhq/selenium-google-code-issue-archive/issues/7723 and https://github.com/SeleniumHQ/selenium/issues/8877 but in the comments I cannot see anything that is actionable.

**Comment by ChristopherLenz** [comment]
Hello, I took a quick look. It has nothing to do with the pictures. In any case, there would be a log message about this beforehand. For me it's the input fields for city and street. These are deactivated and pre-assigned with the last values. This does not validate the script and fails. Should be relatively easy to fix.

```
        #############################
        # set contact street
        #############################
        if ad_cfg["contact"]["street"]:
            self.web_input(By.ID, "pstad-street", ad_cfg["contact"]["street"])
```

**Comment by Talvien** [comment]
Same issue here:
[ERROR] InvalidElementStateException: Message: invalid element state: Element is not currently interactable and may not be manipulated
  (Session info: chrome=101.0.4951.64)

**Comment by DenisLanz** [comment]
This fixes the issue for me, I can prepare a pull request, but need to make myself familiar on how I can contribute…

```python
#############################
        # set contact zipcode
        #############################
        if ad_cfg["contact"]["zipcode"] and self.webdriver.find_element(By.ID, "pstad-zip" ).is_enabled():
            self.web_input(By.ID, "pstad-zip", ad_cfg["contact"]["zipcode"])

        #############################
        # set contact street
        #############################
        if ad_cfg["contact"]["street"] and self.webdriver.find_element(By.ID, "pstad-street").is_enabled():
            self.web_input(By.ID, "pstad-street", ad_cfg["contact"]["street"])
```

**Comment by sebthom** [comment]
@DenisLanz I merged your PR, so the error goes away for now, however what we actually should do is enable the newly introduced checkbox `Vollständige Adresse anzeigen` which will make the street writeable again.

**Comment by DenisLanz** [comment]
> @DenisLanz I merged your PR, so the error goes away for now, however what we actually should do is enable the newly introduced checkbox `Vollständige Adresse anzeigen` which will make the street field writeable again.

Fair point, maybe I give it a try this weekend.

**Comment by sebthom** [comment]
Fixed by #79. Thanks to @DenisLanz
